FileShares/src/FileSharesWeb/Views/Files/Upload.cshtml

```diff
@@ -0,0 +1,40 @@
+@model UploadViewModel
+@{
+    ViewData["Title"] = "Upload Files";
+    int? fileCount = Model?.Files?.Count;
+}
+
+@if (!string.IsNullOrEmpty(Model?.Error))
+{
+    <div class="alert alert-danger" role="alert">
+        <h2 class="alert-heading">There was an error uploading the files</h2>
+        <p>@Model.Error</p>
+    </div>
+}
+@if (Model?.Files?.Count > 0)
+{
+    <div class="alert alert-info" role="alert">
+        <h2 class="alert-heading">Files were uploaded successfully</h2>
+        @foreach ((string originalFilename, string savedFilename) in Model.Files)
+        {
+            string cssStyle = --fileCount == 0 ? "mb-0" : string.Empty;
```

Comment:
Can we just use a simple 2-column grid or table instead of this? It looks hacky, which distracts from the purpose of this sample. I wouldn't want to advocate: this is how you should use Bootstrap.